monit/bin/monit.sh

```diff
@@ -125,17 +138,17 @@ generate(){
       echo "Not yet implemented"
     ;;
   esac  
- }
+}
 
- start(){
+ init(){
```

Comment:
Will renaming this have knock on effects?

Reply:
I don't think so - from what Eoin explained to me, he changed this to allow us to use the start command to start processes rather than just the monit daemon.